unknownObj: inf
person: inf
bicycle: inf
car: inf
motorcycle: inf
train: inf
truck: inf
boat: inf
bridge: inf
buildingOtherMerged: inf
gravel: 5
railroad: inf
road: 0
pavementMerged: 200
dirtMerged: 15
sand: 5
snow: 15
floorOtherMerged: 5
grassMerged: 100
wallOtherMerged: inf
rockMerged: 15
waterOther: 15
treeMerged: inf
fenceMerged: inf
skyOtherMerged: 0
floorConcrete: 0
container: inf
selfArm: inf
selfLeg: 0
stone: inf
gravelPile: inf
sandPile: inf
constructionMachine: inf
bucket: 0
gripper: 0
unknown: 50